bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.html

```diff
@@ -53,13 +52,13 @@
               (selectedEventChange)="selectSequence($event); loadTraces($event.sequence)"
             ></ktb-root-events-list>
             <div fxLayout="row" fxLayoutAlign="center">
-              <ktb-loading-distractor *ngIf="loading" uitestid="keptn-loadingOldSequences"
-                >Loading …</ktb-loading-distractor
-              >
+              <ktb-loading-distractor *ngIf="loading" uitestid="keptn-loadingOldSequences">
+                Loading …
+              </ktb-loading-distractor>
               <button
                 dt-show-more
-                *ngIf="!loading && !project?.allSequencesLoaded"
-                (click)="loadOldSequences()"
+                *ngIf="!loading && !state.sequenceInfo?.allSequencesLoaded"
+                (click)="loadOldSequences(state.projectName)"
```

Comment:
"Show older sequences" is displayed, although no older sequences available.
![image](https://user-images.githubusercontent.com/6098219/175293983-c7aa30bd-db66-401b-8290-2b9267d4076a.png)

Reply:
Fixed. FYI this problem only appeared after the polling happened (the previous value was not considered)